resources/views/checkout/partials/sidebar/user-info.blade.php

```diff
@@ -1,25 +1,27 @@
-<x-rapidez-ct::card v-if="checkout.shipping_address && checkout?.shipping_address?.firstname && checkout.step !== 3">
-    <x-rapidez-ct::title.lg class="mb-4">
-        @lang('Shipping & billing address')
-    </x-rapidez-ct::title.lg>
-    <div class="flex flex-wrap justify-between">
-        <ul class="[&>*+*]:mt-1">
-            <li v-if="checkout?.shipping_address?.firstname && checkout?.shipping_address?.lastname">
-                @{{ checkout?.shipping_address?.firstname + ' ' + (checkout?.shipping_address?.middlename ?? '') + ' ' + checkout?.shipping_address?.lastname }}
-            </li>
-            <li v-if="checkout?.shipping_address?.street[0] && checkout?.shipping_address?.street[1]">
-                @{{ checkout?.shipping_address?.street[0] + ' ' + checkout?.shipping_address?.street[1] + ' ' + (checkout?.shipping_address?.street[2] ?? '') }}
-            </li>
-            <li v-if="checkout?.shipping_address?.postcode && checkout?.shipping_address?.city">
-                @{{ checkout?.shipping_address?.postcode + ' ' + checkout?.shipping_address?.city }}
-            </li>
-            <li v-if="checkout?.shipping_address?.country_id">
-                @{{ checkout?.shipping_address?.country_id }}
-            </li>
-        </ul>
-        <span class="underline text-inactive cursor-pointer self-end" v-if="checkout.step == 2" v-on:click="goToStep(1)" >
-            @lang('Edit')
-        </span>
-    </div>
-</x-rapidez-ct::card>
-
+<template v-if="checkout.step < 4">
+    <template v-if="checkout.hide_billing || checkout.shipping_address?.customer_address_id == checkout.billing_address?.customer_address_id">
+        <x-rapidez-ct::card.gray>
```

Comment:
hmm, card.grey? can you show me where that is used for? i thought we only had 2 different cards? Can you show me in person?